### DISCUSSION TITLE
I can't install appflowy

### DISCUSSION THREAD

**Discussion by torkralle** [AppFlowy-IO/AppFlowy#43] [General help wanted] [answered]
situation:
run ```make install_rust```
then ```make: *** No rule to make target `install_rust'.  Stop.```

I have not found makefile, and I can't install appflowy.
please help me...

**Comment by ghost** [comment, marked as answer]
OS?

Are you running `make install_rust` in the root of the project?

**Reply by torkralle** [reply]
thank you for replying!
OS is Big Sur.
yes, I'm running the command in the root.

**Reply by torkralle** [reply]
Sorry, I got it. 
thanks!
